plenum/server/node.py

```diff
@@ -1856,18 +1995,29 @@ def startViewChange(self, proposedViewNo: int):
         :param proposedViewNo: the new view number after view change.
         """
```

Comment:
This is just of possible strategies for doing view change.
We need to make it independent and pluggable.
We will replace it in future for sure with something more efficient.

If it's too hard to split this logic now, please add TODO comments and create a ticket in Jira for this